src/styles/ProofCenteredLine.css

```diff
@@ -2,10 +2,10 @@
     margin: 5px;
     transition: background-color 0.4s;
     text-overflow: clip;
-    overflow-x: clip;
+    overflow-x: auto;
```

Comment:
Unfortunately, I think overflow-x: auto causes scroll bar to appear:
![image](https://github.com/typefaceoff/typefaceoff/assets/86962800/3a539685-21d1-4c58-a995-c8fe959c15df)

Is it possible to maybe try increasing the max-height instead to solve the clipping problem? Or are we ok with scroll bar

Reply:
![image](https://github.com/typefaceoff/typefaceoff/assets/100410646/fb73f7b5-cd3a-4e15-bee5-d995b5fa02b7)
Looks somewhat better with `visible` instead of `auto`.

Reply:
Changing to FullWidthLine eliminated this issue, and another issue with saving as pdf.

Reply:
Update: changed to custom TitleLine instea.

Reply:
Much better! Thanks for doing that!